{
    "has_execution_error": true,
    "execution_output": "Traceback (most recent call last):\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 65, in <module>\n    main()\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 55, in main\n    proposed_time = find_meeting_time(start_time, end_time, duration, schedules)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 24, in find_meeting_time\n    if block[0] <= time + duration <= block[1]:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<=' not supported between instances of 'int' and 'datetime.datetime'",
    "pred": null,
    "gold": {
        "day": "Monday",
        "time_range": "{11:30:12:00}"
    },
    "status": "Error",
    "violated_constraint": {
        "missing_fields": true
    },
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 1,
    "timing": {
        "api_call_time": 16.31909441947937,
        "execution_time": 0.012337923049926758,
        "constraint_eval_time": 9.5367431640625e-07,
        "pred_extract_time": 2.6226043701171875e-06,
        "gold_extract_time": 1.811981201171875e-05,
        "token_count": 0
    }
}